scripts/install

```diff
@@ -33,6 +33,9 @@ popd
 #=================================================
 # SYSTEM CONFIGURATION
 #=================================================
+
+chown -R $app:www-data "$install_dir/files"
```

Comment:
This permission is already applied to line 19

Reply:
Bonjour, effectivement (j'ai bien vu la ligne 19), mais je crois que suite aux instructions qui suivent le chmod les `bin/console glpi:*` il y a des fichiers qui sont créés avec le propriétaire `root` et dans mon cas cela bloquait l'affichage de la page de login. Je vois que tu as fait du nettoyage/refactoring dans les scripts d'install sur Master. Je vais retester avec ta version.

Reply:
Maybe we should execute the command with `ynh_exec_as_app php$php_version bin/console glpi:config:set`